### ISSUE TITLE
FEATURE: Checkbox for offline use

### ISSUE THREAD

**Issue by Emie510** [rodekruis/helpful-information#1306]
### **Who requested this feature?**
Maaike NLRC

### **User Story**
As a community member 
I want to check a box that confirms I can work offline with this region
So that access the information while I'm offline

### Solution hypothesis
Checkbox on region level that cashes the pages in the app. This downloads all the data in the info. The icons are not loaded, also not the images in the offers. 

### **What needs to be done**
- [ ] Create popup, determine where the button should be
- [ ] While offline, make a difference in UI which region is downloaded offline vs online
- [ ] Create placeholder for the offline mode images.

### **Acceptance Criteria**
- [ ] Info accesible
